### ISSUE TITLE
Add Support for .m3u8 file 

### ISSUE THREAD

**Issue by shoaibhassan01** [eroscai/SZAVPlayer#34]
Can we have it working with .m3u8 file hosted at CDN server?

**Comment by eroscai** [comment]
Yes, it should be supported by default. Are you currently experiencing issues with playback? If so, you can share the link with me, and I can take a look when I have some time.
